base.sbt

```diff
@@ -0,0 +1,2 @@
+addCommandAlias("gitSnapshots", ";set version in ThisBuild := git.gitDescribedVersion.value.get + \"-SNAPSHOT\"")
+enablePlugins(GitBranchPrompt)
```

Comment:
I'm sure because it's purely subjective: I hate having junk in my prompt (sbt-web or play does the same and I hate that too).